implementation.md

```diff
@@ -373,11 +373,15 @@ properly to decide that two actions with the “same” parameters are identical
 Which parameters must be considered for two actions to be identical is entirely
 chosen by the implementer.
 
-When the system is going to perform actions, it sorts them by priority (smaller
-numbers are higher priority). If two actions are going to use the same
+When the system is going to perform actions, it sorts them by how long it has 
+been since the last action, an `Action`'s `priority` (smaller numbers are higher 
```

Comment:
"it sorts them by how long it has been since the last action" I'm not sue I understand this - since the last action of this type was run? Since the action was last checked?

Reply:
Yeah, fair - 'last action' being an action taken on an Action, update() etc. I'll reword this